sdk/core/azure-core/pom.xml

```diff
@@ -152,6 +152,7 @@
       <!-- RestProxyTests is inherited by tests in azure-core-http-netty -->
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
+        <version>3.1.2</version>
```

Comment:
i noticed a couple of plugin versions. is this something enforced in parent/pom.xml

Reply:
Removed version from here.